service/src/test/java/bio/terra/cbas/controllers/TestMethodsApiController.java

```diff
@@ -213,7 +217,7 @@ void returnErrorForInvalidPostRequest() throws Exception {
       }
       """;
     String expectedError =
-        "Bad user request. Error(s): method_name is required. method_source is required and should be one of: [GitHub]. method_version is required. method_url is invalid. Supported URI host(s): [raw.githubusercontent.com]";
+        "Bad user request. Error(s): method_name is required. method_source is required and should be one of: [GitHub, Dockstore]. method_version is required";
```

Comment:
Do we want to add a test for a Github type workflow with an invalid github url to make sure that part of the message is still shown sometimes?

Reply:
I have added another test to cover case of invalid method url with unsupported host